aspnetcore/includes/not-latest-version-without-not-supported-content.md

```diff
@@ -0,0 +1,21 @@
+:::moniker range="< aspnetcore-9.0"
+> [!NOTE]
+> This isn't the latest version of this article. For the current release, see the [.NET 9 version of this article](?view=aspnetcore-9.0&preserve-view=true).
+:::moniker-end
+
+:::moniker range="> aspnetcore-9.0"
+> [!IMPORTANT]
+> This information relates to a pre-release product that may be substantially modified before it's commercially released. Microsoft makes no warranties, express or implied, with respect to the information provided here.
+>
+> For the current release, see the [.NET 9 version of this article](?view=aspnetcore-9.0&preserve-view=true).
+:::moniker-end
+
+<!--
+Include this file at the top of articles. When a new version is released,
+it might be necessary to temporarily comment out the current version
+moniker range section until the new moniker is created.
+Markdown to include this file:
+[!INCLUDE[](~/includes/not-latest-version-without-not-supported-content.md)]
+
+This version of the file doesn't include the not-supported-version content.
```

Comment:
Consider adding a comment explaining when this version needs to be used instead of the original and then add that comment to the original as well.